packages/grafana-ui/src/components/VizTooltip/SeriesTable.tsx

```diff
@@ -14,10 +14,10 @@ export interface SeriesTableRowProps {
   isActive?: boolean;
 }
 
-const getSeriesTableRowStyles = (theme: GrafanaTheme) => {
+const getSeriesTableRowStyles = (theme: GrafanaTheme2) => {
   return {
     icon: css`
-      margin-right: ${theme.spacing.xs};
+      margin-right: ${theme.v1.spacing.xs};
```

Comment:
Instead of using v1 reference, you can use `theme.spacing(factor)` to get the desired value :)

You can read a bit more about the GrafanaTheme2 in this document: https://github.com/grafana/grafana/blob/main/contribute/style-guides/themes.md#paddings-and-margins